src/environment/shell.go

```diff
@@ -401,6 +403,10 @@ func (env *ShellEnvironment) HasFolder(folder string) bool {
 	return hasFolder
 }
 
+func (env *ShellEnvironment) RealPath(path string) (string, error) {
```

Comment:
I would rename this to `ResolveSymlink` as that what this does.

Reply:
renamed